google-cloud-translate/lib/google-cloud-translate.rb

```diff
@@ -62,8 +70,10 @@ module Cloud
     #   translation = translate.translate "Hello world!", to: "la"
     #   translation.text #=> "Salve mundi!"
     #
-    def translate key = nil, retries: nil, timeout: nil
-      Google::Cloud.translate key, retries: (retries || @retries),
+    def translate key = nil, scope: nil, retries: nil, timeout: nil
```

Comment:
Yeah, this is an unfortunate situation. I have two reasons for leaving it this way:

1. Maintaining backwards compatibility. We currently direct users to create translate objects with API Key, and if we changed those arguments to both `project` and `keyfile`, as we have in other services, existing code will break.
2. The design guidelines we've been given state that API Key should have priority over service accounts, as you state.